source/simulator/parameters.cc

```diff
@@ -611,6 +611,14 @@ namespace aspect
                          "as opposed to continuous. This then requires the assembly of face terms "
                          "between cells, and weak imposition of boundary terms for the composition "
                          "field via the discontinuous Galerkin method.");
+      prm.declare_entry ("Use limiter for discontinuous solution", "false",
+                         Patterns::Bool (),
+                         "Whether to apply the bound preserving limiter as a postprocess after having"
+                         "the discontinous solution. Currently we should apply this only to the"
```

Comment:
I would remove the 'should', "Currently we apply this only ..." is more clear on what happens.